packages/auth/package.json

```diff
@@ -26,7 +26,7 @@
     "@graphql-tools/load": "^8.0.1",
     "@graphql-tools/schema": "^10.0.2",
     "@interledger/http-signature-utils": "2.0.0",
-    "@interledger/open-payments": "6.0.0",
+    "@interledger/open-payments": "6.5.0",
```

Comment:
Now that we are bumping to 6.5.0, can you please add `useHttp: process.env.NODE_ENV === 'development'` when we instantiate the Open Payments client please? 

change in question: https://github.com/interledger/open-payments/releases/tag/%40interledger%2Fopen-payments%406.1.0

Reply:
This will also make the Examples > Open Payments Postman folder work

Reply:
After updating this and trying locally, the Grant Requests for Incoming Payment and Quotes work, but the Outgoing payment grant request does not, will need to keep digging

Reply:
That last part should work after https://github.com/interledger/rafiki/pull/2337 is in